docs/SRS/SRS.tex

```diff
@@ -26,20 +29,25 @@ \section*{Revision History}
 \begin{tabularx}{\textwidth}{p{3cm}p{2cm}X}
 \toprule {\bf Date} & {\bf Version} & {\bf Notes}\\
 \midrule
-Oct 2/22 & 1.0 & Copying Volere Template and updating sections\\
+Oct 1/22 & 0.0 & Copying Volere Template and updating sections\\
+Oct 1/22 & 0.1 & Adding Sections 1, 3, 5\\
+
 \bottomrule
 \end{tabularx}
 
 ~\newpage \pagenumbering{arabic}
 
 \section{The Purpose of the Project}
-a
 
 \subsection{The User Business or Background of the Project Effort}
-a
+The business aims to make finding shows across groups of friends with differing preferences easier by streamlining the process from start to finish. As a quick summary, users will be able to input their preferences, make groups with their friends, and the group will get recommendations based on all their preferences. In addition, an opportunity arose in the market after COVID passed as large groups can create host gatherings for any events, one such that this application can select.
 
 \subsection{Goals of the Project}
-a
+Goals can shift as development continues, entirely losing scope on the developers' passion for why the application started. Thus there will only be a few immediate goals to capture the passion that developers currently have and will continue to uphold in the future:
+\begin{itemize}
+	\item Provide a means of choosing a show to watch immediately in a large friend group, which they all enjoy
```

Comment:
replace 'show' with 'movie, tv show, or anime'
we are also supporting individuals as well as groups, should probably mention that